### DISCUSSION TITLE
$:/AdvancedSearch高级搜索，筛选功能的增强建议

### DISCUSSION THREAD

**Discussion by Zacharia2** [tiddly-gittly/TidGi-Desktop#248] [Ideas]

**Comment by linonetwo** [comment]
应该是某个插件加的吧，如果知道了可以加上去

**Reply by linonetwo** [reply]
反正我都用 commander 做这个

**Comment by Zacharia2** [comment]


**Reply by linonetwo** [reply]
那奇怪了，难道是关了某个配置

**Reply by Zacharia2** [reply]
禁用inonetwo/itonnote插件就可以了，应该是这个插件的某些配置

**Reply by Zacharia2** [reply]
而且 ESC 取消编辑 快捷键 似乎也不能用。空白太微是可以用的。我感觉这个快捷键很好用的。

**Reply by linonetwo** [reply]
那个插件预置了很多配置项，估计是其中某一个，要是知道了可以删了

**Reply by Zacharia2** [reply]
> 而且 ESC 取消编辑 快捷键 似乎也不能用。空白太微是可以用的。我感觉这个快捷键很好用的。
> 

原来改成shift-Escape快捷键啦。cancel-edit-tiddler

**Reply by Zacharia2** [reply]
> 那个插件预置了很多配置项，估计是其中某一个，要是知道了可以删了

感觉像用css隐藏了。

**Reply by linonetwo** [reply]
哦对，esc可能是会和 codemirror 的取消提示冲突了。

不过这个快捷键我自己都忘了，估计还是在某种太记手册里间隔重复一下

**Reply by linonetwo** [reply]
> > 那个插件预置了很多配置项，估计是其中某一个，要是知道了可以删了
> 
> 感觉像用css隐藏了。

有可能，因为用css隐藏了默认的搜索框，可能误杀了别的东西

**Reply by Zacharia2** [reply]
![image](https://github.com/tiddly-gittly/TidGi-Desktop/assets/32425955/f93ffacf-77ce-45f1-a71f-90ddec85a165)

删除这个，可以显示按钮，只不过是垂直显示的。

**Reply by linonetwo** [reply]
因为上面有个 flex-direction column

**Reply by Zacharia2** [reply]
![image](https://github.com/tiddly-gittly/TidGi-Desktop/assets/32425955/50d28aed-a890-483e-b01f-157c4785edba)
删掉了，确实是这个CSS文件的修改。

**Reply by Zacharia2** [reply]
不过我不太熟悉这个CSS，我感觉筛选这个地方的按钮还不错，可以保留一下。

**Reply by linonetwo** [reply]
得调整 css 让它仅隐藏侧边栏的搜索框，又不隐藏别的地方的东西。估计得在前面加个侧边栏的css类来限制一下生效范围

**Reply by Zacharia2** [reply]
嗯呢，可以。对了还发现一个问题，就是太记默认知识模版的这个东东，
![image](https://github.com/tiddly-gittly/TidGi-Desktop/assets/32425955/9c08502a-8ff2-43fe-b926-5bc5deed92e0)

我的wiki显示在条目下面，但是太记的官网却正常显示，还有太微中文教程的在太记中显示我我的一样在条目下面遮挡住了，但手机上正常显示，浏览器同太记一样也是遮挡了。

**Reply by Zacharia2** [reply]
![image](https://github.com/tiddly-gittly/TidGi-Desktop/assets/32425955/4a64271d-83f4-4b7f-a74a-8b108bbf1cae)

**Reply by linonetwo** [reply]
估计是你刚刚css改错了吧，之前应该是有 media query 限制窄屏才出现

**Reply by Zacharia2** [reply]
> 估计是你刚刚css改错了吧，之前应该是有 media query 限制窄屏才出现

应该不是这个问题，太微中文教程也有。
